assets/src/domain/eventEditor/services/apollo/queries/tickets/useTicketQueryOptions.ts

```diff
@@ -26,7 +27,8 @@ const useTicketQueryOptions = (datetimeIn: EntityId[] = []): DatetimesQueryOptio
 		},
 	};
 
-	return options;
+	const datetimeInStr = datetimeIn.join(':');
+	return useMemo(() => options, [datetimeInStr]);
```

Comment:
I've created a helper, will extend with this approach as well, after the merge:

https://github.com/eventespresso/event-espresso-core/pull/2862/files#diff-2368976113d2a8bc6cdbeb64364abdb6